bin/grouper-api

```diff
@@ -29,6 +30,34 @@ class Application(tornado.web.Application):
         super(Application, self).__init__(*args, **kwargs)
 
 
+class SendEmailThread(Thread):
+    def run(self):
+        while True:
+            logging.debug("Sending emails...")
+            try:
+                now = datetime.utcnow()
```

Comment:
I'm not sure what you mean... have a function like `send_email(now)` and have the main loop call it with `datetime.utcnow()`?